transport/src/test/java/io/netty/channel/local/LocalChannelTest.java

```diff
@@ -1352,7 +1352,6 @@ public void messageReceived(ChannelHandlerContext ctx, Buffer buffer) throws Exc
                                     // Fill the ChannelOutboundBuffer with multiple buffers
                                     ctx.write(buffer.readSplit(1));
                                 }
-                                buffer.close();
```

Comment:
Why this change ?

Reply:
The `SimpleChannelInboundHandler` releases/closes the message for us, and that would produce an exception if we already closed it here.
This doesn't impact the test, though. It just logs the exception.

Reply:
got it...